Core/Graphics/Buffer.h

```diff
@@ -14,9 +14,9 @@
 #endif
 
 #if defined(CGL_RHI_OPENGL)
-    #include <Core/Graphics/RHI/OpenGL/OPENGLVertexBuffer.h>
-    #include <Core/Graphics/RHI/OpenGL/OPENGLIndexBuffer.h>
-    #include <Core/Graphics/RHI/OpenGL/OPENGLConstantBuffer.h>
+#include <Core/Graphics/RHI/OpenGL/OPENGLVertexBuffer.h>
```

Comment:
Add this to the top of the file. Since `BufferDesc` makes use of the types
![image](https://github.com/user-attachments/assets/3a239fa0-09d4-4051-9096-34205e2f140c)